src/main/java/uk/gov/hmcts/darts/courthouse/CourtroomRepository.java

```diff
@@ -0,0 +1,11 @@
+package uk.gov.hmcts.darts.courthouse;
```

Comment:
Do we think this should be under a `repository` subpackage also, wondering whether this will be `courthouse.repository` or `common.repository` as that is closer to where the Entity class can be found?